env-setup.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+#################
+# Run unpriveleged for vagrant provisioning
+# Don't rely on rsynced folders
+#################
+
+if [ -z "$1" ]; then
+    echo usage: $0 provisioned file directory?
+    echo "using ."
+    TOP_LOC="."
+else
+    TOP_LOC="$1"
+fi
+
+wget "https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh" -O miniconda.sh
```

Comment:
I'm not certain about putting the conda installation steps in this file. I'm imagining that someone new to the project who already has conda installed could use this script to jump into ease development. I think travis is the only place where we want something to run this step all the time.

Reply:
The miniconda install will not attempt another installation if it detects an already existing installation. I can also add a test for a conda installation and bypass this step.